python/test/regression/test_performance.py

```diff
@@ -149,8 +149,8 @@ def test_elementwise(N, dtype_str):
         (4, 48, 4096, 64, True, False, 'forward', 'float16'): 0.306,
         (4, 48, 4096, 64, True, False, 'forward', 'bfloat16'): 0.266,
         (4, 48, 1024, 16, True, False, 'forward', 'float32'): 0.098,
-        (4, 48, 4096, 64, True, False, 'backward', 'float16'): 0.134,
-        (4, 48, 4096, 64, True, False, 'backward', 'bfloat16'): 0.135,
+        (4, 48, 4096, 64, True, False, 'backward', 'float16'): 0.157,
```

Comment:
Why should the perf number be updated?

Reply:
Is what I wrote in the commit message insufficient?  ("This is an improvement to some A100 benchmarks...")

If I improve the benchmarks but don't update the numbers here, the tests fail.